views.py

```diff
@@ -1,9 +1,11 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```

Comment:
nit: These are no longer necessary since we're on Python 3.

Reply:
Nice. Might as well get rid of them https://github.com/dimagi/langcodes/pull/36/commits/f66a486a6b59660349c339e5ab30c68b261fea39